### ISSUE TITLE
After installing VisualVM 2.1.8 on a Mac Apple M1 Pro, the “Perform GC” function cannot be used.

### ISSUE THREAD

**Issue by sititch** [oracle/visualvm#584]
After installing VisualVM 2.1.8 on a Mac Apple M1 Pro, the “Perform GC” function cannot be used.

**Comment by thurka** [comment]
This report is too vague. Please provide steps to reproduce and `messages.log`. Thanks.

**Comment by sititch** [comment]
[logfile.txt](https://github.com/user-attachments/files/16026324/logfile.txt)
Nothing happens when you click the perform gc button


> 此报告太模糊。请提供重现步骤`messages.log`。谢谢。

**Comment by thurka** [comment]
Thanks for the log file, although I still missing steps to reproduce. :-( However the log shows that you have unusual proxy settings. All is set to `127.0.0.1:7890` including socks proxy. The problem is probably caused by setting socks proxy - this setting is used when VisualVM communicates with monitored application and probably this not what you want. Please modify the proxy settings and disable proxy alltogether or at least remove socks proxy.

**Comment by sititch** [comment]
> 感谢日志文件，尽管我仍然缺少重现步骤。:-(但是日志显示您的代理设置不寻常。所有设置为`127.0.0.1:7890`包括 socks 代理。问题可能是由设置 socks 代理引起的 - 此设置在 VisualVM 与受监控应用程序通信时使用，这可能不是您想要的。请修改代理设置并完全禁用代理或至少删除 socks 代理。
<img width="1354" alt="image" src="https://github.com/oracle/visualvm/assets/39956341/2bf5918d-e707-4cb9-900c-6268039ad612">
Thank you so much for taking a look at my log file, but obviously this is not what I wanted. The log file doesn't capture my problem, which is that nothing happens when I click Perform GC.

**Comment by sititch** [comment]
> Thanks for the log file, although I still missing steps to reproduce. :-( However the log shows that you have unusual proxy settings. All is set to `127.0.0.1:7890` including socks proxy. The problem is probably caused by setting socks proxy - this setting is used when VisualVM communicates with monitored application and probably this not what you want. Please modify the proxy settings and disable proxy alltogether or at least remove socks proxy.

Thank you so much for taking a look at my log file, but obviously this is not what I wanted. The log file doesn't capture my problem, which is that nothing happens when I click Perform GC.
<img width="1319" alt="image" src="https://github.com/oracle/visualvm/assets/39956341/ecad2879-d629-470d-a758-3dd256926154">

**Comment by thurka** [comment]
Please follow my advice - change proxy settings, restart VisualVM and attach new log file.

**Comment by sititch** [comment]
<img width="556" alt="image" src="https://github.com/oracle/visualvm/assets/39956341/848c41f3-e5ff-439d-aff3-1d530e32ec84">


> Please follow my advice - change proxy settings, restart VisualVM and attach new log file.

It still doesn't work.
[Uploading logfile.txt…]()

**Comment by thurka** [comment]
I don't see the log file. The link to log file does not work.

**Comment by sititch** [comment]
> I don't see the log file. The link to log file does not work.

[logfile.txt](https://github.com/user-attachments/files/16027915/logfile.txt)

**Comment by thurka** [comment]
The log file now looks ok, provided that you run VisualVM, opened your application and invoked 'Perform GC'.

**Comment by sititch** [comment]
> The log file now looks ok, provided that you run VisualVM, opened your application and invoked 'Perform GC'.

Yes, but I got nothing when I clicked on it, and reinstalling it won't work either

**Comment by sititch** [comment]
> The log file now looks ok, provided that you run VisualVM, opened your application and invoked 'Perform GC'.

There are no other older versions available

**Comment by sititch** [comment]
I clicked on version 2.1.7 and it didn't work

Uploading 录屏2024-07-01 09.44.34.mov…

**Comment by thurka** [comment]
> > The log file now looks ok, provided that you run VisualVM, opened your application and invoked 'Perform GC'.
> 
> Yes, but I got nothing when I clicked on it, 

Based on your screenshot above, it will be hard to see the effect of doing direct GC. BTW: How did you conclude that the 'Perform GC' action does not work?

> and reinstalling it won't work either

I don't see a reason to reinstall it.

**Comment by thurka** [comment]
> > The log file now looks ok, provided that you run VisualVM, opened your application and invoked 'Perform GC'.
> 
> There are no other older versions available

Are you looking for older versions of VisualVM?

**Comment by thurka** [comment]
> I clicked on version 2.1.7 and it didn't work
> 
> Uploading 录屏2024-07-01 09.44.34.mov…

There is no file linked to this comment.

**Comment by sititch** [comment]
> > I clicked on version 2.1.7 and it didn't work
> > Uploading 录屏2024-07-01 09.44.34.mov…
> 
> There is no file linked to this comment.


Uploading 录屏2024-07-01 09.44.34.mov…

**Comment by sititch** [comment]
<img width="437" alt="image" src="https://github.com/oracle/visualvm/assets/39956341/ffabe855-ae7f-4874-a0c5-dba967853c29">
<img width="1139" alt="image" src="https://github.com/oracle/visualvm/assets/39956341/ea323069-fcb9-4998-9cc8-acc4f31ddbb0">
[root@localhost pmce]# java -version
openjdk version "1.8.0_262"
OpenJDK Runtime Environment (build 1.8.0_262-b10)
OpenJDK 64-Bit Server VM (build 25.262-b10, mixed mode)
[root@localhost pmce]# 
I hit visual gc not to display
